src/sonic-config-engine/minigraph.py

```diff
@@ -1003,6 +1008,11 @@ def parse_linkmeta(meta, hname):
             linkmetas[port]["AutoNegotiation"] = auto_negotiation
         if macsec_enabled:
             linkmetas[port]["MacSecEnabled"] = macsec_enabled
+        if tx_power:
+            linkmetas[port]["tx_power"] = tx_power
```

Comment:
tx_power can be float, so is that taken care?

Reply:
Yes, this is taken care